### PR TITLE
Registration confirm callback endpoint

### LINKED ISSUE #87: Add endpoint for registration confirmation

### PR DESCRIPTION
**Pull request by salvanicola**
Fix #87 #85

### CODE REVIEW

File: web-api/src/Kerbero.Identity/Modules/Users/Controllers/UsersController.cs [hunk 1]
```diff
@@ -58,6 +58,14 @@ public async Task<ActionResult<UserReadDto>> Create(UserCreateDto createDto)
     return await _userService.Create(createDto);
   }
 
+  [HttpGet("confirm")]
+  [AllowAnonymous] // TODO ?
```
Comment (vash72) [inline web-api/src/Kerbero.Identity/Modules/Users/Controllers/UsersController.cs:62]:
```suggestion
  [AllowAnonymous]
```

File: web-api/src/Kerbero.Identity/Modules/Users/Controllers/UsersController.cs [hunk 2]
```diff
@@ -58,6 +58,14 @@ public async Task<ActionResult<UserReadDto>> Create(UserCreateDto createDto)
     return await _userService.Create(createDto);
   }
 
+  [HttpGet("confirm")]
```
Comment (vash72) [inline web-api/src/Kerbero.Identity/Modules/Users/Controllers/UsersController.cs:61]:
```suggestion
  [HttpGet("confirm-email")]
```

File: web-api/src/Kerbero.Identity/Modules/Users/Services/UserService.cs [hunk 3]
```diff
@@ -96,6 +98,22 @@ public async Task<UserReadDto> Create(UserCreateDto createDto)
...
 
+  public async Task ConfirmEmailAsync(Guid userId, string code)
+  {
+    var user = await _userManager.GetById(userId);
+    if (user == null)
+    {
+      throw new UnauthorizedException();
+    }
+    
+    code = Encoding.UTF8.GetString(WebEncoders.Base64UrlDecode(code));
```
Comment (vash72) [inline web-api/src/Kerbero.Identity/Modules/Users/Services/UserService.cs:109]:
```suggestion
    code = Encoding.UTF8.GetString(WebEncoders.Base64UrlDecode(code));
```
it is really need to encode/decode the confirm email "code"?

**Review by vash72 — CHANGES_REQUESTED** [review]

File: web-api/src/Kerbero.Identity/Modules/Users/Services/UserService.cs (hunk 3 above)
Comment (salvanicola) [reply web-api/src/Kerbero.Identity/Modules/Users/Services/UserService.cs:109]: The microsoft online example did it. I copied, but not seems to be a bad idea.

**Review by vash72 — APPROVED** [review]

**Pull request merged**